beginners/easy/023.swap-two-number/206.saadman-galib.swap-two-number.py

```diff
@@ -0,0 +1,7 @@
+a, b = map(int, input("Put your input here: ").split())
+
+a = a ^ b
```

Comment:
can you pls add the explanation of this program in python

Reply:
ok bhaiya